monai/data/utils.py

```diff
@@ -423,6 +447,8 @@ def list_data_collate(batch: Sequence):
     data = [i for k in batch for i in k] if isinstance(elem, list) else batch
     key = None
     try:
+        if config.USE_META_DICT:
+            data = pickle_operations(data)  # bc 0.9.0
```

Comment:
thanks, I understand the concern about flexibility, so just added a module variable `PICKLE_KEY_SUFFIX = TraceKeys.KEY_SUFFIX` instead of modifying the collate/decollate interfaces..

Reply:
OK, this solution looks good to me.

Thanks.